hack/build-s2i-dpdk.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+ELAPSED=0
+TIMEOUT=180
+ATTEMPTS=0
+MAX_ATTEMPTS=10
+build_complete=false
+
```

Comment:
use FEATURES_ENVIRONMENT var to actually deploy dpdk first (I think it's better than documenting it is needed to be deployed manually)